perf_test/ode/KokkosODE_RK.cpp

```diff
@@ -353,8 +353,7 @@ int main(int argc, char** argv) {
         run_benchmark_wrapper<Kokkos::DefaultExecutionSpace>, argc, argv)
         ->UseRealTime()
         ->ArgNames({"n", "model"})
-        ->Args({1000, 1})
-        ->Iterations(common_params.repeat);
```

Comment:
I think the intention here was to let `benchmark` decide (when no `--repeat` is provided.